client/app/bundles/KudosApp/components/TabBar.jsx

```diff
@@ -2,34 +2,36 @@ import React, { PropTypes } from 'react'
 import { observer } from 'mobx-react'
 import AppStore from '../stores/AppStore'
 
+const DropDownItem = ({ itemName, currentTab}) => {
+  const classNameForItem = 'nav-link' + (currentTab === itemName ? ' active' : '');
+
+  return <option className={`${classNameForItem} +  kudos-list__tab`} value={itemName}>
+    {itemName}
+  </option>
+};
+
 @observer
 export class TabBar extends React.Component {
+  static itemChanger(event) {
+    AppStore.kudosStore.setCurrentTab(event.target.value)
+  }
   render() {
-    const tabs = ['Recent', 'My Kudos', 'Awarded Kudos']
-
+    const drop_down = ['All Kudos', 'My Kudos', 'Awarded Kudos']
     return (
-      <ul className="kudos-list__nav">
-        {tabs.map(tab => (
-          <Tab
-            key={tab}
-            tabName={tab}
-            currentTab={AppStore.kudosStore.currentTab}
-            clickHandler={() => AppStore.kudosStore.setCurrentTab(tab)}
-          />
-        ))}
-      </ul>
-    )
-  }
-}
-
-const Tab = ({ tabName, currentTab, clickHandler }) => {
-  const classNameForTab = 'nav-link' + (currentTab == tabName ? ' active' : '')
-
-  return (
-    <li className="kudos-list__tab" onClick={clickHandler}>
-      <span className={classNameForTab}>{tabName}</span>
-    </li>
-  )
+    <div className="kudo_list-changer kudos-list">
+    <select onChange={TabBar.itemChanger} className={"drop_down_menu"}>
```

Comment:
String prop values do not need `{}`.  Do you have sarah t's husky precommit hook thing installed and running the code reformatting?